examples/data/sample-geojson.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Can we try to load them with AJAX from https://github.com/uber-common/deck.gl-data/tree/master/examples

Reply:
I could, but then we'd have to deal with async. Is there a benefit to fetching them?

Reply:
Good idea. We should add a mixed FeatureCollection there. I pulled the geojson from here: https://github.com/uber/deck.gl/blob/master/examples/layer-browser/data/sample.geo.json

Reply:
You are welcome to add more data to the deck.gl-data repository, just check first if the data is under a license. If it is your own hand-crafted GeoJSON then it is of course no problem